.circleci/config.yml

```diff
@@ -74,12 +74,19 @@ jobs:
             sudo apt-get update
             sudo apt-get install python3-pip python3-dev
             sudo pip3 install awscli
+      - run:
+          name: Install Sentry CLI
+          command: |
+            npm install @sentry/cli
       - run:
           name: Build
           command: |
             REACT_APP_BACKEND_BASE_URL=$BACKEND_BASE_URL_TEST \
             REACT_APP_SLACK_CLIENT_ID=$SLACK_CLIENT_ID_TEST \
             npm run build
+      - run:
+          name: Post Deploy
+          command: npm run post-deploy
```

Comment:
I notice this is happening _before_ the actual deploy, don't we want it after?

Reply:
Also in keeping with `npm run deploy:test` let's do `npm run post-deploy:test` for this action specifically, and rename in our project and the main apps/package.json accordingly

Reply:
fixed the order, thanks for catching that 😅